### DISCUSSION TITLE
playbook adding a field "maturity level"

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#9772] [Ideas]
Hello,

In the documentation of the sigma rules they have a chapter [Status](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#status), this status can be compared with a maturity level of the rule. 

I tried to sort plays by this maturity status, but it turns out to be very difficult to see all the plays with the same maturity status.

Is it possible to give this maturity status its own field in the playbook view, so you can use the filter option on the top of the view

![Knipsel1](https://user-images.githubusercontent.com/29328171/218247208-8f254fed-d69c-4ec6-94d9-cb06b9e00364.PNG)

Regards
Bart

**Comment by InfosecGoon** [comment]
Thanks for the suggestion, I'll put this in as a feature request.
